src/composables/currencies.ts

```diff
@@ -46,41 +61,55 @@ const initPollingWatcher = createPollingBasedOnMountedComponents(POLLING_INTERVA
 
 export function useCurrencies({
   withoutPolling = false,
+  selectedProtocol = PROTOCOL_AETERNITY, // TODO - remove default value & make the protocol required
 }: UseCurrenciesOptions = {}) {
-  const minTipAmount = computed(() => 0.01 / (currencyRates.value.usd || 1));
+  const minTipAmount = computed(() => 0.01 / (currencyRates.value?.[selectedProtocol].usd || 1));
   const currentCurrencyRate = computed(
-    (): number => currencyRates.value[currentCurrencyCode.value] || 0,
+    (): number => currencyRates.value?.[selectedProtocol]?.[currentCurrencyCode.value] || 0,
   );
   const currentCurrencyInfo = computed(
     (): ICurrency => CURRENCIES.find(({ code }) => code === currentCurrencyCode.value)!,
   );
 
-  async function loadAeternityData() {
+  async function loadCoinsData() {
     try {
-      const aeMarketData = await CoinGecko.fetchCoinMarketData(
-        AE_COINGECKO_COIN_ID,
+      const [aeMarketData, btcMarketData, dogeMarketData] = await CoinGecko.fetchCoinMarketData(
+        [AE_COINGECKO_COIN_ID, BTC_COINGECKO_COIN_ID, DOGE_COINGECKO_COIN_ID],
```

Comment:
I think this should be more dynamic to avoid the need of adding new entries to the arrays whenever we add new protocol to the app. Maybe we need first establish the protocols that actually used by the accounts in the app, and then reuse this to feed the dynamic market data ref.

```
// Before the composable function
const marketData = ref<Record<Protocol, CoinGeckoMarketResponse>>({});

// This could be stored in the accounts composable as I feel we could reuse it
const protocolsInUse: Protocol[] = uniq(accounts.value.map((account) => account.protocol));

// The actual change
marketData.value = CoinGecko.fetchCoinMarketData(protocolsInUse, currentCurrencyCode.value);
```

Just a rough idea.